packages/apps-config/src/endpoints/testingRococo.ts

```diff
@@ -211,6 +211,14 @@ export function createRococo (t: TFunction): EndpointOption {
         providers: {
           TrustBase: 'wss://rococo.trustednodes.net'
         }
+      },
+      {
```

Comment:
Can you just move this alphabetical? (I know, I let a couple sneak in...)

Reply:
I need to move the rest anyway, will include it there.

Reply:
I re-ordered all info correctly